game_components/rooms/basic_room.py

```diff
@@ -0,0 +1,82 @@
+from abc import ABC, abstractmethod  # abstract classes
+import hashlib
+from game_components.entitiies.player import Player
+from game_components.entitiies.mob import Mob
+
+
+class BaseRoom(ABC):
+    __title = None
+    __description = None
+    __linked_rooms = None
+    __mobs = None
+    __players = None
+    __display_char = None
+    __color = None
+    uid = None
+    number = 0
+    __display_x = None
+    __display_y = None
+
+    def __init__(self, _title: str, _display_char: str, _color: (int, int, int), _description: str,
```

Comment:
`(int, int, int)` isn't a valid type hint. `Tuple[int, int, int]` should be used instead.